react/Avatar/index.jsx

```diff
@@ -0,0 +1,80 @@
+import React from 'react'
+import classNames from 'classnames'
+import PropTypes from 'prop-types'
+import palette from '../../stylus/settings/palette.json'
+import styles from './styles'
+
+const nametohex = (name = "") => {
```

Comment:
Maybe `nameToColor` ? At first sight, I thought that you converted the name to an hexadecimal value.